recipe/meta.yaml

```diff
@@ -13,29 +13,36 @@ source:
 build:
   entry_points:
     - filetype=filetype.__main__:main
-  noarch: python
   number: 0
-  script: {{ PYTHON }} -m pip install . -vv
+  script: {{ PYTHON }} -m pip install . --no-build-isolation --no-deps -vv
+  skip: true  # [py<35]
 
 requirements:
   host:
-    - python >3.6
+    - python
     - pip
+    - setuptools
+    - wheel
   run:
-    - python >3.6
+    - python
 
 test:
   imports:
     - filetype
-
+  source_files:
+    - tests
+  requires:
+    - pytest
+    - pip
   commands:
-    - filetype --help
```

Comment:
is `filetype --help` not working? the entry point is there. It should work

Reply:
It does work. I figured it was unnecessary since we're running the actual test suite, but I'll put it back.